build/changelog/build-changelog.php

```diff
@@ -4,8 +4,14 @@
 
 use Aws\Build\Changelog\ChangelogBuilder;
 
-$changelogBuilder = new ChangelogBuilder("", "", true);
+$params = [
+    'verbose_flag' => true
+];
+
+$changelogBuilder = new ChangelogBuilder($params);
```

Comment:
Since the builder allows users to customize its behavior from the constructor, I think the CLI should accept those inputs from the command line. It sounds like this script will currently only work if run from the correct folder, which means automation scripts would need to `cd` into the right place before executing this command and then `cd` out afterwards.

Ideally, the CLI should honor `-v`, `--outputDir`/`-o`, and `--inputDir`/`-i`.